drivers/bluetooth/hci/ipm_stm32wb.c

```diff
@@ -338,6 +339,20 @@ static void start_ble_rf(void)
 
 	/* Switch OFF LSI */
 	LL_RCC_LSI1_Disable();
+#else
+	LL_RCC_LSI2_Enable();
+	while (!LL_RCC_LSI2_IsReady()) {
+	}
+
+	/* Select wakeup source of BLE RF */
+	LL_RCC_SetRFWKPClockSource(LL_RCC_RFWKP_CLKSOURCE_LSI);
+	LL_RCC_SetRTCClockSource(LL_RCC_RTC_CLKSOURCE_LSI);
+
+	/* Switch OFF LSE */
+	LL_RCC_LSE_Disable();
+	LL_RCC_LSE_DisableBypass();
```

Comment:
I don't this it is needed to switch off LSE in this case anymore.
Hasen't been enabled here prevously, so if enabled, it might be by other component that needs it

Reply:
Actually, you're right! I was thinking about managing the LSE in term if power consumption. But it's not BLE driver responsibility. LSE operations are deleted. Thank you pushing this idea